config-spi/src/main/java/io/airlift/spi/config/InvalidConfigurationResolverException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.spi.config;
+
+public class InvalidConfigurationResolverException
```

Comment:
How about `ConfigurationResolverException`? This sounds like the resolver itself is invalid, but this is simplify a failure to resolve configuration.